src/components/WorkflowSelection.jsx

```diff
@@ -198,15 +198,15 @@ WorkflowSelection.defaultProps = {
 const mapStateToProps = state => {
   const user = state.login.user;
   const userHasWorkInProgress = user && WorkInProgress.check(user);
-
+  
   return {
     //Does the user currently have a page being actively annotated, (e.g. user
     //navigated away from the Classifier page and wants to return), or have
     //saved work in progress? (e.g. user reloaded the website after a crash)
     //We need to know if the user has any work that can be retrieved (either
     //from the Redux store of local storage) so we can prompt them to continue.
     activeAnnotationExists: (!!state.workflow.data && !!state.subject.currentSubject) || userHasWorkInProgress,
-    adminMode: state.login.adminMode,
+    adminMode: false,
```

Comment:
Is there a reason this was changed to false?

Reply:
D'OH!!!